src/aat/java/uk/gov/hmcts/ccd/userprofile/befta/UserProfileBeftaMain.java

```diff
@@ -1,11 +1,12 @@
 package uk.gov.hmcts.ccd.userprofile.befta;
 
 import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
 
 public class UserProfileBeftaMain extends BeftaMain {
 
     public static void main(String[] args) {
-        setTaAdapter(new UserProfileTestAutomationAdapter());
+        setTaAdapter(new DefaultTestAutomationAdapter());
```

Comment:
You don't need to set this adapter. Can just remove this line.
if you would like to use a custom adapter here is the way to do so:
public class DataStoreBeftaMain {

    public static void main(String[] args) {
        BeftaMain.main(args, new DataStoreTestAutomationAdapter());
    }

}